wagtail/admin/wagtail_hooks.py

```diff
@@ -247,3 +256,198 @@ def register_core_features(features):
         WhitelistRule('ul', allow_without_attributes),
         WhitelistRule('li', allow_without_attributes),
     ])
+
+    # Draftail
+    features.register_editor_plugin(
+        'draftail', 'hr', draftail_features.BooleanFeature('enableHorizontalRule')
+    )
+    features.register_converter_rule('contentstate', 'hr', {
+        'from_database_format': {
+            'hr': HorizontalRuleHandler(),
+        },
+        'to_database_format': {
+            'entity_decorators': {'HORIZONTAL_RULE': lambda props: DOM.create_element('hr')}
+        }
+    })
+
+    features.register_editor_plugin(
+        'draftail', 'h1', draftail_features.BlockFeature({
+            'label': 'H1',
+            'type': 'header-one',
+            'description': str(_('Heading {level}').format(level=1)),
```

Comment:
It's a bit silly to use the lazy translation and cast it immediately to a string.